### PR TITLE
fixes russian helmets not holding vodka

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: fixes russian helmets not holding both a vodka and a glass
/:cl:

### CODE REVIEW

**Comment by europaisch** [comment]
quality pr, товарищ

**Comment by europaisch** [comment]
Also this fixes #44686

**Comment by europaisch** [comment]
Also this fixes #44686

File: code/datums/components/storage/concrete/pockets.dm [hunk 1]
```diff
@@ -85,10 +85,14 @@
 	return original_parent
 
 /datum/component/storage/concrete/pockets/small/helmet
-	max_items = 1
+	max_items = 2
```
Comment (nemvar) [inline code/datums/components/storage/concrete/pockets.dm]: I'm sensing some russian bias here.

**Review by 81Denton — CHANGES_REQUESTED** [review]
Document your changes properly por favor

File: code/datums/components/storage/concrete/pockets.dm [hunk 2]
```diff
@@ -84,11 +84,13 @@
 	// if the component is reparented to a jumpsuit, the items still go in the protector
 	return original_parent
 
-/datum/component/storage/concrete/pockets/small/helmet
-	max_items = 1
+/datum/component/storage/concrete/pockets/helmet
 	quickdraw = TRUE
+	max_combined_w_class = 100
```
Comment (ninjanomnom) [inline code/datums/components/storage/concrete/pockets.dm]: You don't need this much

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**